### PR TITLE
Add README for rotate-tokens

### PR DESCRIPTION
**Pull request by cmurphy**

### CODE REVIEW

**Comment by cmurphy** [comment]
@dkeightley @dnoland1 @macedogm

File: rotate-tokens/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This script is used to rotate the main service account and token for a Rancher
+downstream cluster. It may be used in the event of a known token exposure or as
+a routine preemptive measure.
+
+## Usage
+
+Generate an API token in Rancher and use it to set the TOKEN environment
+variable. Set KUBECONFIG to point to your Rancher local cluster. Set
+RANCHER_SERVER to point to your Rancher service. The script can be run without
```
Comment (macedogm) [inline rotate-tokens/README.md:11]:
```suggestion
`RANCHER_SERVER` to point to your Rancher server. The script can be run without
```

File: rotate-tokens/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# rotate-tokens.sh
+
+This script is used to rotate the main service account and token for a Rancher
+downstream cluster. It may be used in the event of a known token exposure or as
+a routine preemptive measure.
+
+## Usage
+
+Generate an API token in Rancher and use it to set the TOKEN environment
```
Comment (macedogm) [inline rotate-tokens/README.md:9]:
```suggestion
Generate an API token in Rancher and use it to set the `TOKEN` environment
```

File: rotate-tokens/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+variable. Set KUBECONFIG to point to your Rancher local cluster. Set
```
Comment (macedogm) [inline rotate-tokens/README.md:10]:
```suggestion
variable. Set `KUBECONFIG` to point to your Rancher local cluster kubeconfig file. Set
```

File: rotate-tokens/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+any arguments. Example:
+
+```
+export TOKEN=token-ccabc:xyz123
+export KUBECONFIG=/path/to/kubeconfig
+export RANCHER_SERVER=https://rancher.example.com
+./rotate-tokens.sh
+```
+
+For extra debugging information, run with DEBUG=y:
```
Comment (macedogm) [inline rotate-tokens/README.md:21]:
```suggestion
For extra debugging information, run with `DEBUG=y`:
```

**Review by macedogm — APPROVED** [review]
I made only a few small comments, otherwise LGTM.

**Review by dkeightley — APPROVED** [review]

**Pull request merged**